### PR TITLE
docs: add focus-visible documentation

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
As we implemented :focus-visible, we need to add the documentation, in general:

* Speak in general terms when to use focus vs. focus-visible
* Show which components use focus and which use focus-visible
* Mention and link to the polyfill needed for implementing focus-visible

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Brad Paugh, David Becher, or Drew Chandler.

### CODE REVIEW

File: docs/getting-started/accessibility/best-practices.html [hunk 1]
```diff
@@ -106,12 +106,46 @@ <h4>
...
+    <li>Any element which supports keyboard input (such as an input element, or any other element which may trigger a virtual keyboard to be shown on focus if a physical keyboard is not present) should always match :focus-visible when focused.</li>
+    <li>If the user interacts with the page via the keyboard, the currently focused element should match :focus-visible (i.e. keyboard usage may change whether this pseudo-class matches even if it doesn’t affect :focus).</li>
+    <li>If the user interacts with the page via a pointing device, such that the focus is moved to a new element which does not support user input, the newly focused element should not match :focus-visible.</li>
+    <li>If the active element matches :focus-visible, and a script causes focus to move elsewhere, the newly focused element should match :focus-visible.</li>
+    <li>Conversely, if the active element does not match :focus-visible, and a script causes focus to move elsewhere, the newly focused element should not match :focus-visible.</li>
```
Comment (braddialpad) [inline docs/getting-started/accessibility/best-practices.html:127]: Very nice, did you write this yourself?

**Review by braddialpad — CHANGES_REQUESTED** [review]
We'll need to add in documentation for the utility classes that support the new fv: class. 

For example, from background color page we currently have this:
Screen Shot 2022-01-17 at 8 47 09 AM

We'll have to add a new one for focus-visible that says something like:
Use fv:d-fc-{color} to change an element's background color only when it is focused by keyboard.

File: docs/getting-started/accessibility/best-practices.html (hunk 1 above)
Comment (juliodialpad) [reply docs/getting-started/accessibility/best-practices.html:127]: No 😅, I put the source in the links below. I thought it was good material to add it there.

Comment (braddialpad) [reply docs/getting-started/accessibility/best-practices.html:127]: > No 😅, I put the source in the links below. I thought it was good material to add it there.

yeah, either way, it's nice haha

**Review by braddialpad — APPROVED** [review]
Looks fantastic, thanks!

**Pull request merged**